pkg/clusterapi/config/containerd_config_append.toml

```diff
@@ -1,10 +1,12 @@
 [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."public.ecr.aws"]
-    endpoint = ["https://{{.registryMirrorAddress}}"]
+    endpoint = ["https://{{index .registryMirrorAddressMappings "public.ecr.aws"}}"]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."783794618700.dkr.ecr.*.amazonaws.com"]
+    endpoint = ["https://{{index .registryMirrorAddressMappings "783794618700.dkr.ecr.*.amazonaws.com"}}"]
```

Comment:
same comment here hardcoding the `783794618700` account

Reply:
same as above